repository/o.json

```diff
@@ -1136,6 +1136,18 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "Ollama AI Sublime",
+			"details": "https://github.com/KalimeroMK/ollama-sublime",
+			"readme": "https://raw.githubusercontent.com/KalimeroMK/ollama-sublime/master/README.md",
```

Comment:
```suggestion
```

Since this is a default path to the readme there is no need to specify it.

Reply:
Thanks. Please add your entry in the correct alphabetical order. Also, packages should not have the word “Sublime” in them: all packages here are for Sublime.